From 25d06f42568a8e627ae0a7f94161ea682f49b9d7 Mon Sep 17 00:00:00 2001
From: Xing Zheng <zhengxing@rock-chips.com>
Date: Sat, 21 Apr 2018 22:45:29 +0800
Subject: [PATCH] arm64: dts: rockchip: add headphone detection by codec for
 RK3308 EVB

This patch using codec detect headphone via simple-card on
RK3308 EVB, it needs add the default widgets "Headphones"
for registering soc-jack.

Change-Id: I9819aa05e8d186d32d9363cfe619782783c67dd0
Signed-off-by: Xing Zheng <zhengxing@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi b/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
index b7af7fa81785..48f6c75346e5 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
@@ -107,6 +107,9 @@
 		simple-audio-card,format = "i2s";
 		simple-audio-card,name = "rockchip,rk3308-acodec";
 		simple-audio-card,mclk-fs = <256>;
+		simple-audio-card,codec-hp-det;
+		simple-audio-card,widgets =
+			"Headphone", "Headphones";
 		simple-audio-card,cpu {
 			sound-dai = <&i2s_8ch_2>;
 		};
-- 
2.35.3

